testing/trino-testing/src/main/java/io/trino/testing/AbstractTestEngineOnlyQueries.java

```diff
@@ -4115,35 +4116,35 @@ public void testCorrelationSymbolMapping()
                 "AND EXISTS (SELECT * FROM T c WHERE b.name = c.name)");
 
         assertQuery("WITH T AS ( " +
-                "SELECT name, min(regionkey) AS key " +
+                "SELECT name, min(regionkey) AS \"key\" " +
```

Comment:
not sure we want to apply delimited identifiers for Trino
what abou  changing to sth like key_column?

Reply:
Reverted actually.